### DISCUSSION TITLE
That one line that save the world

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#69] [General (Memes)]
![image](https://user-images.githubusercontent.com/97792861/164706195-6bfd83f0-c859-44e0-8cf3-64d058eefa8a.png)
This line revived shaders on my engine like W T F

**Comment by ghost** [comment]
What makes this funnier is that this is discussion #69.

**Reply by Theoyeah** [reply]
bruh LOL
